src/controllers/expenses.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const expenseService = require('./../services/expenses');
+const userService = require('./../services/users');
+
+const getAll = (req, res) => {
+  const filters = req.query;
+  const filteredExpenses = expenseService.getAll(filters);
+
+  res.send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseService.getOne(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const create = (req, res) => {
+  const data = req.body;
+  const { name, userId } = req.body;
```

Comment:
not agree, because I use this data(there are not only two params) in function bellow in 44 line `expenseService.create(data)`.
```
const newExpense = {
    id: new Date().valueOf(),
    ...data,
  };
```
Why I separate destructed two params here? I did extra condition exsactly for them